### PR TITLE
feat(action-popover): add horizontalAlignment prop

### PR DESCRIPTION
**Pull request by grabkowski**
### Proposed behaviour
Add horizontalAlignment prop to horizontally align Action Popover Item content.
Screenshot 2021-06-14 at 10 33 11

### Checklist

- [x] Commits follow our style guide
- [x] Screenshots are included in the PR if useful
~~- [ ] All themes are supported if required~~
- [x] Unit tests added or updated if required
- [ ] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [x] Typescript `d.ts` file added or updated if required
- [x] Carbon implementation and Design System documentation are congruent

### Testing instructions
1. npm run
2. open http://localhost:9001/?path=/story/design-system-action-popover--with-item-content-aligned-to-right

### CODE REVIEW

**Review by nicktitchmarsh — DISMISSED** [review]

File: src/components/action-popover/action-popover-menu/action-popover-menu.component.js [hunk 1]
```diff
@@ -164,6 +166,8 @@ ActionPopoverMenu.propTypes = {
   onClick: PropTypes.func,
   /** @ignore @private */
   placement: PropTypes.oneOf(["bottom", "top"]),
+  /** Horizontal alignment of menu items content */
+  horizontalAlignment: PropTypes.oneOf(["left", "right"]),
```
Comment (mmaciorowski-sage) [inline src/components/action-popover/action-popover-menu/action-popover-menu.component.js:170]: Can you update `d.ts` file please?

File: src/components/action-popover/action-popover-item/action-popover-item.component.js [hunk 2]
```diff
@@ -292,6 +298,8 @@ const propTypes = {
   placement: PropTypes.oneOf(["bottom", "top"]),
   /** @ignore @private */
   focusItem: PropTypes.bool,
+  /** Horizontal alignment of menu items content */
+  horizontalAlignment: PropTypes.oneOf(["left", "right"]),
```
Comment (mmaciorowski-sage) [inline src/components/action-popover/action-popover-item/action-popover-item.component.js:302]: Can you update `d.ts` file please?

**Review by mmaciorowski-sage — CHANGES_REQUESTED** [review]
Please update `d.ts` files.

File: src/components/action-popover/action-popover-menu/action-popover-menu.component.js (hunk 1 above)
Comment (grabkowski) [reply src/components/action-popover/action-popover-menu/action-popover-menu.component.js:170]: it's an internal prop, passed from the parent component

**Review by mmaciorowski-sage — DISMISSED** [review]

**Review by mmaciorowski-sage — DISMISSED** [review]

**Review by nicktitchmarsh — DISMISSED** [review]

**Review by nicktitchmarsh — APPROVED** [review]

**Review by mmaciorowski-sage — APPROVED** [review]

**Review by ZhuoyuJin — APPROVED** [review]

**Pull request merged**